### PR TITLE
Adding discord

### PR DESCRIPTION
**Pull request by mrbusysky**
fixes: #1111 

Added a link to discord that doesn't expire.

### CODE REVIEW

File: src/components/Icon/index.tsx [hunk 1]
```diff
@@ -87,6 +88,7 @@ export enum IconType {
   trophy,
   twitter,
   youtube,
+  discord,
```
Comment (thesanjeevsharma) [inline src/components/Icon/index.tsx:91]: This should be in alpabetical order.

File: src/components/Icon/index.tsx [hunk 2]
```diff
@@ -42,6 +42,7 @@ import ThumbsUpIcon from 'mdi-react/ThumbsUpIcon';
 import TrophyIcon from 'mdi-react/TrophyIcon';
 import TwitterIcon from 'mdi-react/TwitterIcon';
 import YoutubeIcon from 'mdi-react/YoutubeIcon';
+import DiscordIcon from 'mdi-react/DiscordIcon';
```
Comment (thesanjeevsharma) [inline src/components/Icon/index.tsx:45]: This should be in alpabetical order.

File: src/components/Icon/index.tsx [hunk 3]
```diff
@@ -186,6 +188,8 @@ const Icon = forwardRef<HTMLDivElement, ComponentProps>(
           return <TwitterIcon {...iconProps} />;
         case IconType.youtube:
           return <YoutubeIcon {...iconProps} />;
+        case IconType.discord:
```
Comment (thesanjeevsharma) [inline src/components/Icon/index.tsx:191]: Here too!

**Comment by thesanjeevsharma** [comment]
@mrbusysky Can you create an issue for this PR first? Every PR needs to be linked with an issue.

File: src/components/Icon/index.tsx (hunk 2 above)
Comment (mrbusysky) [reply src/components/Icon/index.tsx:45]: This was corrected

File: src/components/Icon/index.tsx (hunk 3 above)
Comment (mrbusysky) [reply src/components/Icon/index.tsx:191]: was corrected

**Comment by mrbusysky** [comment]
https://github.com/thenewboston-developers/Website/issues/1111

File: src/components/Icon/index.tsx (hunk 1 above)
Comment (mrbusysky) [reply src/components/Icon/index.tsx:91]: this was corrected already

File: src/utils/social-media.ts [hunk 4]
```diff
@@ -21,6 +23,7 @@ export const socialMediaHandles = {
 };
 
 export const socialMediaDescriptions = {
+  [SocialMedia.discord]: 'Join our community on discord to chat.',
```
Comment (jamessspanggg) [inline src/utils/social-media.ts:26]: Would `Join our discord community to make new friends and interact with one another.` be good?

**Review by jamessspanggg — COMMENTED** [review]
Is it possible to use a non white discord icon? seems blended with the background
Screen Shot 2020-12-12 at 9 19 55 PM

File: src/containers/Openings/OpeningDetails/index.tsx [hunk 5]
```diff
@@ -15,6 +15,7 @@ const customLinks = {
   [SocialMedia.slack]: '',
   [SocialMedia.twitter]: '',
   [SocialMedia.youtube]: '',
+  [SocialMedia.discord]: 'https://discord.gg/fdj2EMvSY',
```
Comment (buckyroberts) [inline src/containers/Openings/OpeningDetails/index.tsx:18]: I notice this is different than the other Discord URL https://github.com/thenewboston-developers/Website/blob/6d5b602fe86d6d9bfaf309e68a63980d546a35a4/src/utils/social-media.ts#L4 Should one of these be used over the other?

**Comment by buckyroberts** [comment]
This marketing card is slightly more narrow than the other ones.

screencapture-localhost-3000-social-2020-12-13-18_25_45

File: src/containers/Openings/OpeningDetails/index.tsx (hunk 5 above)
Comment (mrbusysky) [reply src/containers/Openings/OpeningDetails/index.tsx:18]: done

File: src/utils/social-media.ts (hunk 4 above)
Comment (mrbusysky) [reply src/utils/social-media.ts]: done

File: src/containers/Openings/OpeningDetails/index.tsx (hunk 5 above)
Comment (mrbusysky) [reply src/containers/Openings/OpeningDetails/index.tsx]: corrected, it was a messed up copy paste.

File: src/utils/social-media.ts (hunk 4 above)
Comment (jamessspanggg) [reply src/utils/social-media.ts]: @mrbusysky it seems like the logo is still white

**Comment by jamessspanggg** [comment]
@mrbusysky I have updated the discord logo from their official site located [here](https://discord.com/branding).

**Review by jamessspanggg — APPROVED** [review]
LGTM

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**